src/styles/blocks/card.css

```diff
@@ -0,0 +1,14 @@
+body {
+  margin: 0;
+  font-size: 12px;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+.fullCard {
```

Comment:
css file name is `card.css` and class name is `.fullCard` both names should be same

Reply:
done